acestep/gradio_ui/events/generation_handlers.py

```
@@ -935,46 +999,139 @@ def update_audio_components_visibility(batch_size):
+    elif is_custom:
+        strength_label = t("generation.codes_strength_label")
+        strength_info = t("generation.codes_strength_info")
+    else:
+        # Extract, Lego, Complete — use generic cover strength label
+        strength_label = t("generation.cover_strength_label")
+        strength_info = t("generation.cover_strength_info")
+    strength_update = gr.update(visible=show_strength, label=strength_label, info=strength_info)
+
+    # Think checkbox: requires LLM initialization AND not in Remix/Repaint mode
+    lm_initialized = llm_handler.llm_initialized if llm_handler else False
+    think_interactive = lm_initialized and not (is_cover or is_repaint)
```

Ruff (0.15.0)

[error] Local variable `think_interactive` is assigned to but never used

Remove assignment to unused variable `think_interactive`

(F841)